src/P3Scheme.jl

```diff
@@ -288,6 +231,26 @@ function DSD_N₀(p3::PSP3, N::FT, λ::FT) where {FT}
     return N / Γ(1 + μ) * λ^(1 + μ)
 end
 
+"""
+    integrate(a, b, c1, c2, c3)
+
+ - a - lower bound 
+ - b - upper bound 
+ - c1, c2, c3 - respective constants 
+
+ Integrates the function c1 * D ^ (c2) * exp(-c3 * D) dD from a to b 
+    Returns the result
+"""
+function integrate(a::FT, b::FT, c1::FT, c2::FT, c3::FT) where {FT}
```

Comment:
This function should have a more descriptive name, since it integrates a specific function.